presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -217,6 +217,19 @@ Property Name                                      Description
                                                    format or the default Presto format?
 
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
+                                                   If ``true`` then setting
+                                                   ``hive.insert-existing-partitions-behavior`` to ``APPEND``
+                                                   is not allowed.
+                                                   This also affects ``insert_existing_partitions_behavior``
```

Comment:
Missing word "the". Let's also expand the note

> This also affects the xxx session property in the same way.

Apologies, I should have provided the exact text before.

Reply:
done